c2corg_ui/templates/waypoint/associated.html

```diff
@@ -7,13 +7,13 @@
          ng-style="document['main_waypoint_id'] === waypoint['document_id'] && { 'order' : '-1'}  ">
       
       <%include file="../waypoint/card.html"/>
-      <div translate ng-if="document.main_waypoint_id === waypoint.document_id">main waypoint</div>
+      <div translate ng-if="document.main_waypoint_id === waypoint.document_id" ng-init="document['main_waypoint_title'] = waypoint.locales[0].title">main waypoint</div>
```

Comment:
Using ``document.main_waypoint_title`` instead of ``document['main_waypoint_title']`` would be more consistent with the following lines.